.landscaper/gardenlet/blueprint.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+jsonSchema: "https://json-schema.org/draft/2019-09/schema" # required
+
+# imports are filled via the Installation CRD specifying an import section.
+imports:
```

Comment:
Shouldn't this also list the `apiVersion`/`kind` properties?

Reply:
No, I do not think so as we tested it successfully like this.
cc @schrodit

Reply:
Hm, ok, and why not?

Reply:
@schrodit I guess this is due to the internal design of the landscaper?

Reply:
do you mean that the imports should list the apiVersion and kind?

The imports have no apiVersion or kind as their structure is defined with the jsonschema.

Reply:
Well, the example file (https://github.com/gardener/gardener/pull/3130/files/528ff8838a736ebcce5b32b295e0c059a4adf999#diff-02d0762f8546a5c2825e9f8bbe534e9ef7455d982ebc8d290dbdc233e84cf838) contains:

```yaml
apiVersion: imports.gardenlet.landscaper.gardener.cloud/v1alpha1
kind: LandscaperGardenletImport
...
```

So, I guess, the JSON schema should also contain these fields, or?